### ISSUE TITLE
Bug: template literals are being ignored

### ISSUE THREAD

**Issue by ivan-aksamentov** [edvardchen/eslint-plugin-i18next#20]
The plugin correctly processes single-quoted and double-quoted strings, such as:

```js
'Hello' // warning
"Hello" // warning
```

but wrongly ignores template literals:

```js
`Hello` // no warning
```

Template literals are often allowed in internationalization packages and even if not, they might as well be equivalent to normal strings (i.e. to not contain substitution).

Please consider adding the processing of template literals.

**Comment by edvardchen** [comment]
I knew that. I was afraid that would be too harsh to not allow users using string templates

**Comment by ivan-aksamentov** [comment]
@edvardchen Yes, in fact some programmers prefer to always use template literals. This way it's easier to add substitutions later.

Are they currently *not included* or is there any particular reason to *exclude* them?

**Comment by edvardchen** [comment]
I just don't do any thing about them. I can validate the templates that contain only literal strings no   substitutions

**Comment by andriystabryn** [comment]
hello guys,
how can I validate strings in `backtick` quotes ?

**Comment by edvardchen** [comment]
@andriystabryn I'm not sure if we should validate the template strings. Here is what I am thinking:

1. If the template string has no placeholders, why not just use quotes instead ? Then the plugin would validate them.
2. if it contains placeholders, the user know what he/she is doing and the plugin should not make trouble.

**Comment by andriystabryn** [comment]
@edvardchen 
1. Agree
2. for example we have string 
```js
`I live in ${city}`
```
Also in our case we have a lot of string (huge app) and now we are integrating translations, therefore we need to validate all old string and notify us where we have to replace them. So would be nice if we can enable error for backtick or if you can help and say how to modify your plugin so I can create a patch for us.

Thank in advance!

**Comment by edvardchen** [comment]
OK. It seems to be important for projects that didn't consider internationality when startup.

I will add another option called `validateTemplate` to tell the plugin to validate those backticks

@andriystabryn

**Comment by andriystabryn** [comment]
@edvardchen Thank you very much! will wait for a new release!

**Comment by edvardchen** [comment]
@andriystabryn take a look at test cases. See if these are sufficient

**Comment by andriystabryn** [comment]
@edvardchen for me it works well! When do you plan to merge the PR and release a new version?

**Comment by andriystabryn** [comment]
Hi again @edvardchen , 
unfortunately I've found a bug. I can't add a rule to ignore strings with backticks. Also I've tried to add className to ignoreAttributes and didn't help me :(
```
<div className={`media-screen ${dragClassName}`}> 
```
Also, what I've found is that CONSTANT also is validating 
```
const CONST_STRING = `dsa${test}dsa`;
```
Therefore I guess that backtrics validations skips all kind of ignore rules.

**Comment by edvardchen** [comment]
u r right. We had not cover these cases. I will look up the related code to figure out how to fix that tomorrow

**Comment by edvardchen** [comment]
@andriystabryn it's not easy to find a way to reuse those ignore rules that were applied on strings for template strings. I need to do some research. Welcome PR if someone finds a elegant way.

**Comment by andriystabryn** [comment]
bad news :(
But I'm hopping you will find a solution ;)

Good luck!

**Comment by edvardchen** [comment]
@andriystabryn try the pre-version 5.0.0-0

**Comment by andriystabryn** [comment]
it works, at least at first sight. Will continue testing in real life. 
Thanks!

**Comment by edvardchen** [comment]
because i refactor whole rule so it need to be testing in real world although it passed all test cases
